### PR TITLE
Announce 1.13.1

### PR DESCRIPTION
**Pull request by dawidwys**

### CODE REVIEW

**Comment by knaufk** [comment]
Is it expected that the list of issues still contains OPEN issues?

**Comment by dawidwys** [comment]
Right, it should not... I guess for a minor release it is better to move the open issues to the next version manually. I will take care of this.

**Comment by dawidwys** [comment]
I updated the PR and removed the open issues. I will update the versions in JIRA when releasing 1.13.1.

**Review by zentol — APPROVED** [review]

File: _posts/2021-06-01-release-1.13.1.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+authors:
+- dwysakowicz:
+  name: "Dawid Wysakowicz"
+  twitter: "dwysakowicz"
+
+---
+
+The Apache Flink community released the first bugfix version of the Apache Flink 1.13 series.
+
+This release includes 82 fixes and minor improvements for Flink 1.13.1. The list below includes a detailed list of all fixes and improvements.
```
Comment (rmetzger) [inline _posts/2021-06-01-release-1.13.1.md]: When I search this page for `browse/FLINK-`, I have 52 hits, not 82.

```suggestion
This release includes 52 fixes and minor improvements for Flink 1.13.1. The list below includes a detailed list of all fixes and improvements.
```

**Review by rmetzger — APPROVED** [review]
besides a minor counting issue +1 to merge

File: _posts/2021-06-01-release-1.13.1.md (hunk 1 above)
Comment (dawidwys) [reply _posts/2021-06-01-release-1.13.1.md]: According to some internal discussions, I removed all documentation and test instabilities from the list below. The full release notes can be accessed in JIRA. I probably should've changed the wording here as well. WDYT about:
> The list below includes most important changes. For full list see: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12350058

**Pull request closed without merging**